src/mame/drivers/neptunp2.cpp

```diff
@@ -6,8 +6,91 @@
+ ||1||1||| |27C4001_-_SOUND_|   |OKI   |   |S1 DIPS|              |
+ || ||5|P9 __________________   |MSM6376   |_______|  _________   |
+ ||_||_|   |    27C8000 or  |   |______|   _________  PAT063/31 (PAL16L8)
+ |         |27C4001_-_SOUND_|              |S2 DIPS|              |
+ |__                                                              |
+ ||P|      __________________  __________________    ____    BATT |
+ ||7|      |27C801 or       |  |RAM MS62256-79  |    8583P   3V6  |
+ ||_|      |27C4001_________|  |________________|           179mAh|
+ |__       __________________  __________________                 |
+ ||P|__    |27C801 or       |  |MS628512        |    ____         |
+ ||1||P|   |27C4001_________|  |NOT_POPULATED___|   X24C16        |
+ ||1||18              _______                                     |
+ || |__               |CPLD |    ________    ____________         |
+ ||_||P|              |PD18 |    | 75189 |   | CIRSA     |     __ |
+ |   |17              |_____|  NOT POPULATED | 38302 or  |     |P||
+ |   |_|     XTAL 36.8640MHz     ________    | 38304     |     |2||
+ |__                             | 75188 |   |           |     | ||
+ ||P|      ___________         NOT POPULATED |___________|     | ||
+ ||3|      |CPU       |      ________          ________   ____ |_||
+ ||_|      |80C188XL  |      |7407___|         |7406___| LM393    |
+ |         |          |      ________          ________           |
+ |__       |          |      |74HC14_|         |74HC00_|          |
+ ||P|      |__________|      ________          ________           |
+ ||8|        ________        |74HC14_|         |74HC14_|          |
+ || |        |74HC14_|       ________                             |
+ || |      ____________      |74HCT08|       ____________         |
+ ||_| __   | CIRSA     |     ________        | CIRSA     |        |
+ |    |P|  | 38302 or  |     |74HCT14|       | 38302 or  |        |
+ |__  |1|  | 38304     |     ________        | 38304     |        |
+ ||P| |3|  | MASTER    |     |74HCT14|       |           |        |
+ ||5| __   |___________|                     |___________|        |
+ || | |P|                                                         |
+ || | |4|    ________        ________           ________          |
+ ||_| | |    |ULN2064        |ULN2064           |74LS145 <- NOT POPULATED
+ |    | |    ________        ________         __________          |
+ |    | |    |ULN2064        |ULN2064         |UDN2580A| <- NOT POPULATED
+ |    |_|   __________       ________         __________          |
+ |          |___P14___|      |ULN2064         |_ARRAY__| <- NOT POPULATED
+ |                          NOT POPULATED                 ______  |
+ |_________________________________________________________P19____|
+ 
+P4, P8, P13, P15, P16 and P19 are unused.
+
+Some service manuals contains the comlpete PCB schematics (e.g., see the "Manual Técnico Cirsa Vulcano" PDF).
```

Comment:
Thanks :)
Small typo here: 'comlpete' instead of 'complete'

Reply:
Fixed, thanks!